spring-boot/admin/src/main/resources/static/dashboard/css/gateway-health-monitor.css

```diff
@@ -0,0 +1,55 @@
+.neutral-state-symbol{
+    height: 20px;
+    width: 20px;
+    background-color: grey;
+    border-radius: 50%;
+    display: inline-block;
+    border:1px solid black;
```

Comment:
I think the border would fit in better optically, if it is dark blue like the font in the health monitor.

Reply:
Done